schemas/IDLs/org.gel.models.cva.avro/1.1.0/DataIntake.avdl

```diff
@@ -53,6 +53,29 @@ protocol DataIntakeProtocol {
         clinical
     }
 
+    enum Category {
+        HundredK,
```

Comment:
Are set on this naming ? Could it be 100KGP like I've seen in other places ?

Reply:
Avro does not like numbers at the beginning of an enum.......